ui/components/app/flask/snap-version/snap-version.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
```

Comment:
Adding storybook for this component might be useful in the future. Also a basic unit test would be useful for maintenance.

Reply:
Agreed